playwright/tests/customers.spec.ts

```diff
@@ -0,0 +1,305 @@
+  customersPage = new CustomersPage(page);
+  giftCardsPage = new GiftCardsPage(page);
+  addressesListPage = new AddressesListPage(page);
+  addressForm = new AddressForm(page);
+  addAddressDialog = new AddAddressDialog(page);
+  deleteAddressDialog = new DeleteAddressDialog(page);
+});
+
+test("TC: SALEOR_199 Create customer @e2e @customer", async () => {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const note = faker.lorem.sentence();
+  const email = faker.internet.email();
+  await customersPage.goToCustomersListView();
+  await customersPage.clickOnCreateCustomer();
+  await customersPage.fillFirstAndLastName(firstName, lastName);
+  await customersPage.fillEmail(email);
+  const newAddress = ADDRESS.addressUS;
+  await addressForm.typeFirstName(newAddress.firstName);
+  await addressForm.typeLastName(newAddress.lastName);
+  await addressForm.typeCompanyName(newAddress.companyName);
+  await addressForm.typePhone(newAddress.phone);
+  await addressForm.typeAddressLine1(newAddress.addressLine1);
+  await addressForm.typeAddressLine2(newAddress.addressLine2);
+  await addressForm.typeCity(newAddress.city);
+  await addressForm.typeZip(newAddress.zip);
+  await addressForm.selectCountry(newAddress.country);
+  await addressForm.waitForDOMToFullyLoad();
+  await addressForm.selectCountryArea(newAddress.countryArea);
+  await customersPage.fillNote(note);
+  await customersPage.saveCustomer();
+  await customersPage.expectSuccessBanner();
+  await expect(customersPage.pageHeader).toContainText(
+    `${firstName} ${lastName}`,
+  );
+  await expect(customersPage.customerNoteInput).toContainText(note);
+  await expect(customersPage.customerFirstNameInput).toHaveValue(firstName);
+  await expect(customersPage.customerLastNameInput).toHaveValue(lastName);
+  await expect(customersPage.customerEmailInput).toHaveValue(
+    email.toLowerCase(),
+  );
+});
+test("TC: SALEOR_200 As an admin I should not be able to create customer with duplicated email @e2e @customer", async () => {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const note = faker.lorem.sentence();
+  await customersPage.goToCustomersListView();
+  await customersPage.clickOnCreateCustomer();
+  await customersPage.fillFirstAndLastName(firstName, lastName);
+  await customersPage.fillEmail(CUSTOMERS.customerToBeDeactivated.email);
+  const newAddress = ADDRESS.addressUS;
+  await addressForm.typeFirstName(newAddress.firstName);
+  await addressForm.typeLastName(newAddress.lastName);
+  await addressForm.typeCompanyName(newAddress.companyName);
+  await addressForm.typePhone(newAddress.phone);
+  await addressForm.typeAddressLine1(newAddress.addressLine1);
+  await addressForm.typeAddressLine2(newAddress.addressLine2);
+  await addressForm.typeCity(newAddress.city);
+  await addressForm.typeZip(newAddress.zip);
+  await addressForm.selectCountry(newAddress.country);
```

Comment:
If this part repeats ( or a smaller chunk of it ) in other tests maybe it is worth considering creating a helper in addressForm Page

Reply:
@wojteknowacki so should I add fillAddressFormAllFields(address: AddressFieldsType)  **back** now?

Reply:
completeBasicInfoAddressForm() added